### PR TITLE
Form creation

### PR DESCRIPTION
**Pull request by kmasouri**
:neckbeard:

### CODE REVIEW

File: views/home.hbs [hunk 1]
```diff
@@ -36,30 +37,101 @@
...
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col s12">
+                                    <input type="checkbox" id="legal">
+                                    <label for="legal">Are you legally authorized to hold a paid off-campus
+                                        internship in the U.S.?</label>
+                                </div>
+                            </div>
+                            <div class="divider" style="margin:16px"></div>
```
Comment (MArmstrong95) [inline views/home.hbs]: I didn't know how to add this part to the css file. But there are several dividers on this page, just with 16px margins.

File: views/home.hbs [hunk 2]
```diff
@@ -36,30 +37,101 @@
...
                     <form method="post">
                         <div class="card-content">
-                            <div class="card-tabs">
-                                <ul class="tabs tabs-fixed-width">
-                                    <li class="tab"><a class="active" href="#step1">Step 1</a></li>
-                                    <li class="tab"><a href="#step2">Step 2</a></li>
-                                    <li class="tab"><a href="#step3">Step 3</a></li>
-                                </ul>
+                            <span class="card-title">Internship Agreement: Student Information</span>
+                            <div class="divider" style="margin:16px"></div>
```
Comment (MArmstrong95) [inline views/home.hbs]: inline styled divider

File: views/home.hbs [hunk 3]
```diff
@@ -36,30 +37,101 @@
...
+                            <div class="row">
+                                <div class="input-field col s12">
+                                    <textarea id="textarea2" class="materialize-textarea" name="textbox2"></textarea>
+                                    <label for="textarea2">What professional and personal goals do you hope to achieve
+                                        while at this internship? (Be specific)</label>
                                 </div>
-                                <div id="step3">
-                                    Step 3
+                            </div>
+                            <div class="divider" style="margin: 16px"></div>
```
Comment (MArmstrong95) [inline views/home.hbs]: inline styled divider

File: views/home.hbs [hunk 4]
```diff
@@ -71,20 +143,330 @@
...
+    {{#compare userType '==' 'supervisor'}}
+    <!--Card form for internship site supervisor-->
+    <div class="container">
+        <div class="row valign-wrapper center-align card-container">
+            <div class="col s12">
+                <div class="card">
+                    <form method="post">
+                        <div class="card-content">
+                            <span class="card-title">Internship Agreement: Internship Site Information</span>
+                            <div class="divider" style="margin:16px"></div>
```
Comment (MArmstrong95) [inline views/home.hbs]: inline style

File: views/home.hbs [hunk 5]
```diff
@@ -71,20 +143,330 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="input-field col s12">
+                                    <input type="text" id="additional_comp" name="additionalComp">
+                                    <label for="additional_comp">Additional Compensation/Stipend</label>
+                                </div>
+                            </div>
+                            <div class="divider" style="margin: 16px"></div>
```
Comment (MArmstrong95) [inline views/home.hbs]: styled div

File: views/home.hbs [hunk 6]
```diff
@@ -71,20 +143,330 @@
...
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="input-field col s12">
+                                        <textarea id="site_textarea4" class="materialize-textarea"
+                                                  name="siteTextarea4"></textarea>
+                                    <label for="site_textarea4">Additional comments regarding internship</label>
+                                </div>
+                            </div>
+                            <div class="divider" style="margin: 16px"></div>
```
Comment (MArmstrong95) [inline views/home.hbs]: styled div

File: views/home.hbs [hunk 7]
```diff
@@ -71,20 +143,330 @@
...
+    {{#compare userType '==' 'faculty'}}
+    <!--Faculty card for signature and input-->
+    <div class="container">
+        <div class="row valign-wrapper center-align card-container">
+            <div class="col s12">
+                <div class="card">
+                    <form method="post">
+                        <div class="card-content">
+                            <span class="card-title">Internship Agreement: Faculty Liaison</span>
+                            <div class="divider" style="margin:16px"></div>
```
Comment (MArmstrong95) [inline views/home.hbs]: styled div

File: views/home.hbs [hunk 8]
```diff
@@ -7,7 +7,8 @@
     <ul id="slide-out" class="side-nav fixed">
         <li class="logo" align="center">
             <a id="logo-container" href="/home" class="brand-logo">
-                <img id="front-page-logo" src="https://upload.wikimedia.org/wikipedia/en/2/28/Winthrop_University_seal.svg" height="100%">
+                <img id="front-page-logo"
+                     src="https://upload.wikimedia.org/wikipedia/en/2/28/Winthrop_University_seal.svg" height="100%">
```
Comment (cschipani) [inline views/home.hbs]: height inline style

**Comment by cschipani** [comment]
@MArmstrong95, approve your review and we can merge it.

**Pull request merged**